.github/workflows/wheels.yml

```diff
@@ -195,10 +179,18 @@ jobs:
             PIP_EXTRA_INDEX_URL=https://pypi.anaconda.org/scientific-python-nightly-wheels/simple
             PIP_NO_BUILD_ISOLATION=false
 
+      - name: Rename after test (macOS x86 Accelerate only)
```

Comment:
Instead of renaming back, can you perhaps copy the original wheel to a new dir (renaming to `13_0` in the process) and then discard it? Saves a step, and it feels a bit nicer to ship the `14_0.whl` without manually having had to touch it.

Reply:
I'm currently taking advantage of the fact that `cibuildwheel` uses glob matching to get the repaired wheel. This allows me to perform the rename as part of the repair command, and `cibuildwheel`will proceed to install and test that wheel. After the tests complete, `cibuildwheel` copies this repaired, renamed wheel to the `wheelhouse` folder, at which point I rename it back.

Alternatively, we can have `cibuildwheel` skip its tests and add a new step to copy, rename, install, and test the wheel. What do you think?

Reply:
Thanks for the explanation. It was just a suggestion and I didn't have the full context here; if this is easier then let's keep it as it is now.